Update linux-resources.md

andrewjohnporter opened the pull request:
Hi, apologies, this isn't actually a change for this page but I am unaware how to create a completely new page. I would like this same page for for Linux troubleshooting instead. Specifically I had an issue with no license found on Linux. This is the text there should be for that new section:

If Microsoft Defender for Endpoint on Linux has been offboarded When the offboarding script is executed on the Linux, it saves a file in /etc/opt/microsoft/mdatp and it's named mdatp_offboard.json.

If the file exists, it will prevent the Linux Server from being onboarded again. Delete the mdatp_offboard.json then run the onboarding script again.

This is the link for MACs, I would like one created for Linux Servers: https://learn.microsoft.com/en-us/defender-endpoint/mac-support-license

andrewjohnporter committed:
Update linux-resources.md

Hi, apologies, this isn;t actually a change for this page but I am unaware how to create a completely new page. I would like this same page for for Linux troubleshooting instead. Specifically I had an issue with no license found on Linux. This is the text there should be for that new section:

If Microsoft Defender for Endpoint on Linux has been offboarded
When the offboarding script is executed on the Linux, it saves a file in /etc/opt/microsoft/mdatp and it's named mdatp_offboard.json.

If the file exists, it will prevent the Linux Server from being onboarded again. Delete the mdatp_offboard.json then run the onboarding script again.

This is the link for MACs, I would like one created for Linux Servers:
https://learn.microsoft.com/en-us/defender-endpoint/mac-support-license

denisebmsft commented:
Hello @andrewjohnporter! Thank you for the feedback. I have added a work item in our queue for this request. I'm going to close this pull request but please know that we are looking into this. BTW, feel free to email me at deniseb@microsoft.com. My team manages documentation for Defender for Endpoint and other security offerings. I have seen several pull requests and such from you with helpful feedback. Thank you for this one, and the previous ones.

andrewjohnporter commented:
Thank you. Appreciate all the docs you guys do.

________________________________
From: Denise Vangel-MSFT ***@***.***>
Sent: 19 February 2025 5:03 PM
To: MicrosoftDocs/defender-docs ***@***.***>
Cc: Andrew John Porter ***@***.***>; Mention ***@***.***>
Subject: Re: [MicrosoftDocs/defender-docs] Update linux-resources.md (PR #196)


Hello @andrewjohnporter<https://github.com/andrewjohnporter>! Thank you for the feedback. I have added a work item in our queue for this request. I'm going to close this pull request but please know that we are looking into this. BTW, feel free to email me at ***@***.******@***.***>. My team manages documentation for Defender for Endpoint and other security offerings. I have seen several pull requests and such from you with helpful feedback. Thank you for this one, and the previous ones.

—
Reply to this email directly, view it on GitHub<https://github.com/MicrosoftDocs/defender-docs/pull/196#issuecomment-2669239618>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AMWWHH6ZYGLMSW3WQKPHMT32QS2NPAVCNFSM6AAAAABXJY27M2VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMNRZGIZTSNRRHA>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

[denisebmsft]denisebmsft left a comment (MicrosoftDocs/defender-docs#196)<https://github.com/MicrosoftDocs/defender-docs/pull/196#issuecomment-2669239618>

Hello @andrewjohnporter<https://github.com/andrewjohnporter>! Thank you for the feedback. I have added a work item in our queue for this request. I'm going to close this pull request but please know that we are looking into this. BTW, feel free to email me at ***@***.******@***.***>. My team manages documentation for Defender for Endpoint and other security offerings. I have seen several pull requests and such from you with helpful feedback. Thank you for this one, and the previous ones.

—
Reply to this email directly, view it on GitHub<https://github.com/MicrosoftDocs/defender-docs/pull/196#issuecomment-2669239618>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AMWWHH6ZYGLMSW3WQKPHMT32QS2NPAVCNFSM6AAAAABXJY27M2VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDMNRZGIZTSNRRHA>.
You are receiving this because you were mentioned.Message ID: ***@***.***>
